data/rest-data.go

```diff
@@ -106,6 +108,8 @@ func (r *RestData) Setup() error {
 	r.token = r.Config.GetString("token")
 
 	_ = r.getMetadata()
+	r.getTopDirContents()
+	r.getForgeDirContents()
```

Comment:
Hoisted the fetch of this data out of the security insights func so we can read it once and then use for dealing with other commons repository files as well.